data/World/Deku Tree MQ.json

```
@@ -19,8 +19,8 @@
                 here(can_use(Slingshot) or can_use(Bow)) and
                 here(has_fire_source_with_torch or can_use(Bow))",
             "Deku Tree Basement Water Room Front": "
-                here(can_use(Slingshot) or can_use(Bow)) and here(has_fire_source_with_torch)",
-            "Deku Tree Basement Ledge": "logic_deku_b1_skip or here(is_adult)"
+                (here(can_use(Slingshot) or can_use(Bow)) and here(has_fire_source_with_torch))",
```

This added paren doesn't seem meaningful.
```suggestion
                here(can_use(Slingshot) or can_use(Bow)) and here(has_fire_source_with_torch)",
```